### PR TITLE
Get rid of /extension/ folder

### PR DESCRIPTION
**Pull request by KillariDev**
fix https://github.com/DarkFlorist/TheInterceptor/issues/159

### CODE REVIEW

File: Dockerfile
Comment (MicahZoltu) [inline Dockerfile:39]: I'm assuming this was a file move plus adjusting all of the paths?  No code changes otherwise?

File: test/tests/signTypedData.ts
Comment (MicahZoltu) [inline test/tests/signTypedData.ts:2]: How did this not fail CI previously?  Sounds like a bug in our CI process?

**Review by MicahZoltu — APPROVED** [review]

File: Dockerfile
Comment (KillariDev) [reply Dockerfile:39]: yea, I removed `extension/` from multiple places

File: test/tests/signTypedData.ts
Comment (KillariDev) [reply test/tests/signTypedData.ts:2]: It did fail the CI. I forgot to wait for CI to complete before merging (as CI is not run to branches not merging to main). I wonder if we could get it to run for those too?

Comment (KillariDev) [reply test/tests/signTypedData.ts:2]: I tried to create release and then I noticed it was failing for this: <https://github.com/DarkFlorist/TheInterceptor/actions/runs/4858826351>

**Pull request merged**
